### PR TITLE
Propose config changes to replace env with more structured chain parameters

### PR DESCRIPTION
**Pull request by chrisli30**
The src/config/index.js will act as a chain config selector for us to handle more and more chains. Using env is good for the short-term but not really scalable.

This PR is an example of how to use it in src/shibuya.js. The demo is working with my changes.

### CODE REVIEW

File: src/common/shibuyaHelper.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
 import { ApiPromise, WsProvider } from '@polkadot/api';
-import { instructionWeight, env } from './constants';
+import _ from 'lodash';
+import { instructionWeight } from './constants';
```
Comment (chrisli30) [inline src/common/shibuyaHelper.js:3]: Note that `env` is deprecated from this file.

File: src/common/shibuyaHelper.js [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
 import { getProxyAccount } from './utils';
-
-const { SHIBUYA_PARA_ID } = env;
+import { Shibuya } from '../config';
```
Comment (chrisli30) [inline src/common/shibuyaHelper.js:5]: Everything about ss58 and paraId should be in this `Shibuya`. The same goes to `TuringDev`.

For example, 
```
import { MangataDev, TuringDev } from '../config';
```

File: src/config/mangata-dev.js [hunk 3]
```diff
@@ -0,0 +1,200 @@
+const assets = [
```
Comment (chrisli30) [inline src/config/mangata-dev.js:1]: This config needs to be updated later.

These are the assets I grabbed from Mangata, but their dev environment has fewer parameters.

**Review by imstar15 — APPROVED** [review]
Okay, looks good to me!

**Pull request merged**